test/unit/cinstance_test.rb

```diff
@@ -683,6 +683,12 @@ class ChangePlanTest < ActiveSupport::TestCase
       end
       assert_includes @cinstance.errors['plan_id'], 'not allowed in this context'
     end
+
+    test 'cannot change plan to not plan at all' do
+      previous_plan = @cinstance.plan_id
+      assert_nothing_raised { @cinstance.change_plan! nil }
```

Comment:
This is not what I meant 😝 This is what I meant: https://github.com/3scale/porta/pull/580
Feel free to either merge it or close it or to do what you consider with it... but it is easier to do it and explain it in the description there than adding a big abstract comment here.

Reply:
BTW `assert_nothing_raised` is completely useless :)